scopes/component/component/ui/use-component-query.ts

```diff
@@ -110,11 +118,15 @@ const SUB_COMPONENT_REMOVED = gql`
 `;
 
 /** provides data to component ui page, making sure both variables and return value are safely typed and memoized */
-export function useComponentQuery(componentId: string, host: string) {
+export function useComponentQuery(
+  componentId: string,
+  host: string,
+  filters?: { logType?: string; logOffset?: number; logLimit?: number; logHead?: string }
```

Comment:
these filters are for an inner field of the component. 
the arg name should reflect it.
otherwise, I don't know what these filters going to affect

Reply:
fixed. Changed the arg to `filters?: { logFilters?: { logType?: string; logOffset?: number; logLimit?: number; logHead?: string }}`